courses/views_test.py

```diff
@@ -188,33 +196,34 @@ def test_duplicate_post_doesnt_create(self):
         """
         If we post twice with the same course_id, it should trigger an update.
         """
-        resp = self.client.post(reverse('course-list'), {
-            "title": "title1",
-            "author_name": "author1",
-            "overview": "overview1",
-            "description": "description1",
-            "course_id": COURSE_ID,
-            "image_url": "/234.jpg",
-            "instructors": [
-                "861e87a0803e436b989cb62d5e672c5f",
-                "961e87a0803e436b989cb62d5e672c5f"
-            ]
-        })
-        self.assertEqual(resp.status_code, 201, resp.content)
-        resp = self.client.post(reverse('course-list'), {
-            "title": "title1",
-            "author_name": "author1",
-            "overview": "overview1",
-            "description": "description1",
-            "course_id": COURSE_ID,
-            "image_url": "/234.jpg",
-            "instructors": [
-                "861e87a0803e436b989cb62d5e672c5f",
-                "961e87a0803e436b989cb62d5e672c5f"
-            ]
-        })
-        self.assertEqual(resp.status_code, 200, resp.content)
-        assert Course.objects.count() == 1
+        with mock.patch('courses.views.module_population', autospec=True):
+            resp = self.client.post(reverse('course-list'), {
+                "title": "title1",
+                "author_name": "author1",
+                "overview": "overview1",
+                "description": "description1",
+                "course_id": COURSE_ID,
+                "image_url": "/234.jpg",
+                "instructors": [
+                    "861e87a0803e436b989cb62d5e672c5f",
+                    "961e87a0803e436b989cb62d5e672c5f"
+                ]
+            })
+            self.assertEqual(resp.status_code, 201, resp.content)
+            resp = self.client.post(reverse('course-list'), {
```

Comment:
Can you remove this code from the context block? Since it's an update it shouldn't need `module_population`, and removing it would help verify that it wasn't called